packages/operator/pkg/apiserver/routes/v1/connection/connection_validation_test.go

```diff
@@ -83,6 +83,22 @@ func (s *ConnectionValidationSuite) TestEmptyURL() {
 	s.g.Expect(err.Error()).Should(ContainSubstring(conn_route.EmptyURIErrorMessage))
 }
 
+
+func (s *ConnectionValidationSuite) TestEmptyURLEcr() {
+	conn := &connection.Connection{
+		Spec: v1alpha1.ConnectionSpec{
+			Type:      connection.EcrType,
+			Reference: connReference,
+			KeySecret: creds,
+		},
+	}
+	err := s.v.ValidatesAndSetDefaults(conn)
+	s.g.Expect(err).Should(HaveOccurred())
```

Comment:
I remember we discussed that having 2 testing frameworks (gomega + testify) is redundant and further tests we can write with testify. So that we could easier throw gomega away some day.

Reply:
Yes, I agree, but it's a small fix